plugins/inputs/logstash/logstash.go

```diff
@@ -0,0 +1,290 @@
+package logstash
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	jsonparser "github.com/influxdata/telegraf/plugins/parsers/json"
+)
+
+const sampleConfig = `
+  ## This plugin reads metrics exposed by Logstash Monitoring API.
+  # https://www.elastic.co/guide/en/logstash/current/monitoring.html
```

Comment:
Remove this comment from the sample config, this info will be covered by the description string and the readme.